src/Package/Impl/DataInspect/Commands/ImportDataSetTextFileCommand.cs

```diff
@@ -17,7 +17,10 @@ internal sealed class ImportDataSetTextFileCommand : SessionCommand {
         protected override void Handle() {
             base.Handle();
 
-            string filePath = VsAppShell.Current.BrowseForFileOpen(IntPtr.Zero, Package.Resources.CsvFileFilter);
+            string filePath = VsAppShell.Current.BrowseForFileOpen(
```

Comment:
None of the other commands that need VsAppShell have it passed in that I can see, so I guess I'll change all of them.